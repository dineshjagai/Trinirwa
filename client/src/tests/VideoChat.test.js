import React from 'react';
import VideoChat from '../live_stream/VideoChat';
import { createMount } from '@material-ui/core/test-utils';

test('Test Menu snapshot', () => {
  const component = createMount(<VideoChat />);
  expect(component).toMatchSnapshot();
});

