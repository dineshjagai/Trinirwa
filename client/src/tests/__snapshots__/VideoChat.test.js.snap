// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Menu snapshot 1`] = `[Function]`;
